src/spyglass/behavior/v1/moseq.py

```diff
@@ -308,6 +308,84 @@ def get_training_progress_path(self, key: dict = None):
         )
         return f"{project_dir}/{model_name}/fitting_progress.pdf"
 
+    def generate_trajectory_plots(
+        self, key: dict = True, output_dir: Path = None, **kwargs
+    ):
+        """calls the moseq function to generate the pose trajectory of each syllable
+
+        Parameters
+        ----------
+        key : dict, optional
+            restriction to the moseq model table, by default True
+        output_dir : Path, optional
+            where to save the figure to, default of None saves to project_dir
+
+        Returns
+        -------
+        None
+        """
+        self.ensure_single_entry(key)
+        query = self & key
+        project_dir = (query).fetch1("project_dir")
```

Comment:
Minor. could consolidate the fetch:
`project_dir, model_name = (self & key).fetch1("project_dir", "model_name")`